lightning.services.yml

```diff
@@ -44,3 +44,10 @@ services:
     arguments: []
     tags:
       - { name: drupal.generator }
+
+  lightning.lightning_version:
```

Comment:
This service ID is kinda ambiguous. Maybe lightning.version_command would be clearer?